lib/node_modules/@stdlib/_tools/changelog/generate/README.md

```diff
@@ -0,0 +1,110 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2024 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# Generate Changelog
+
+> Generate a changelog for a specified package.
```

Comment:
```suggestion
> Generate a changelog for a specified stdlib package.
```

This package is implemented (and intended to be) very `stdlib` aware. So we should clarify that this is specifically for generating a stdlib package changelog here and elsewhere.